sympy/vector/coordsysrect.py

```diff
@@ -98,6 +213,65 @@ def __new__(cls, name, location=None, rotation_matrix=None,
             location = Vector.zero
             origin = Point(name + '.origin')
 
+        if transformation is None:
+            if parent is None:
+                if rotation_matrix is not None:
+                    rotation_equations = \
+                       lambda x, y, z: CoordSys3D._rotation_trans_equations(parent_orient, lambda x, y, z: (x, y, z))
+                else:
+                    rotation_equations = lambda x, y, z: (x, y, z)
+
+                # Information about location without parent doesn't have sense.
+                # See lines 80:99. Location is somehow treated only when parent is put.
+                # We are not able to create independent vector, so we can't assign
+                # anything to location.
+                translation_equations = lambda x, y, z: (x, y, z)
```

Comment:
I believe we should treat the parent as being a default standard coordinate system if it is not specified. Otherwise what sense does it make to specify the transformation?